### DISCUSSION TITLE
Custom tooltips for buttons (aria-labels)

### DISCUSSION THREAD

**Discussion by AHTOHO4** [igordanchenko/yet-another-react-lightbox#300] [Q&A] [answered]
Where can I see a full list with the names of all the buttons for custom hover hints?

I only found a couple:
1) Previous
2) Next
3) Close

But I'm also interested in the ** Enter Fullscreen, Exit Fullscreen, ZoomIn, ZoomOut** buttons

p.s This is how I indicate aria label

```tsx
<Lightbox
  labels={{ 
    Previous: t('Common.prevTitle'), 
    Next: t('Common.nextTitle'), 
    Close: t('Common.closeTitle') 
  }}
/>
```

**Comment by igordanchenko** [comment]
You are bringing up a good point. There is no such list at the moment, and I will look into improving this DX aspect. For now, you'd need to use a web inspector to obtain the labels.

<img width="625" alt="Screenshot 2024-08-08 at 8 37 28 AM" src="https://github.com/user-attachments/assets/0bc654ea-75dd-4029-add1-d46f24fc5343">

**Reply by igordanchenko** [reply, marked as answer]
You can now use IntelliSense code-completion with v3.21.3 to obtain a list of all applicable labels (the list depends on the plugins you use)

<img width="579" alt="Screenshot 2024-08-08 at 8 53 15 AM" src="https://github.com/user-attachments/assets/a46ccaf5-e40c-4ac8-b903-c741c5f4ad15">

**Comment by AHTOHO4** [comment]
Thank you, I appreciate your work. You solved my problem.
